server/StudySharp.API/Requests/TheoryBlocks/GetTheoryBlocksRequest.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+using StudySharp.Domain.Models;
+
+namespace StudySharp.API.Requests.TheoryBlocks
+{
+    public class GetTheoryBlocksRequest
```

Comment:
add CourseId

Reply:
Lmao
We do not need this file, so i deleted it